float32 cpuUsage
uint32 availableMemory
